src/config.ts

```diff
@@ -214,7 +215,10 @@ export function resolveConnectionsFromEnv(): ConnectionConfig[] {
       database: process.env.DB_NAME,
       options: {
         encrypt: process.env.DB_ENCRYPTION === 'true'
-      }
+      },
+      // Knex config for mssql uses these keys in place of "host" and "user"
+      server: process.env.DB_SERVER,
```

Comment:
Let's still use DB_HOST as the environment variable to avoid confusion? Also, does server and userName keys work for postgres and mssql.

```suggestion
      server: process.env.DB_HOST,
```

Reply:
Yes, that's the confusion I had. Should we handle this change in the library or let the user handle it? No, postgres still requires `host` and 'user`. This is specifically for mssql. Don't know why Knex did this. 🤷

Reply:
Let's still use DB_HOST as our variable. This is under our control and is less confusing.